base/tests/utils/integration/com/google/idea/blaze/base/ExternalWorkspaceFixture.java

```diff
@@ -0,0 +1,77 @@
+package com.google.idea.blaze.base;
+
+import com.google.idea.blaze.base.ideinfo.ProjectDataInterner;
+import com.google.idea.blaze.base.lang.buildfile.psi.BuildFile;
+import com.google.idea.blaze.base.lang.buildfile.references.LabelUtils;
+import com.google.idea.blaze.base.model.BlazeProjectData;
+import com.google.idea.blaze.base.model.primitives.ExternalWorkspace;
+import com.google.idea.blaze.base.model.primitives.Label;
+import com.google.idea.blaze.base.model.primitives.TargetName;
+import com.google.idea.blaze.base.model.primitives.WorkspacePath;
+import com.google.idea.blaze.base.model.primitives.WorkspaceRoot;
+import com.google.idea.blaze.base.sync.data.BlazeProjectDataManager;
+import com.intellij.psi.PsiFile;
+
+import javax.annotation.Nullable;
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.idea.blaze.base.settings.ui.ProjectViewUi.getProject;
+import static org.junit.Assert.assertNotNull;
+
+public class ExternalWorkspaceFixture {
+  public final ExternalWorkspace w;
+
+  final TestFileSystem fileSystem;
+  WorkspaceFileSystem workspaceFileSystem;
+
+  public ExternalWorkspaceFixture(ExternalWorkspace w, TestFileSystem fileSystem) {
+    this.w = w;
+    this.fileSystem = fileSystem;
+  }
+
+  public WorkspaceRoot getWorkspaceRoot() {
```

Comment:
It is unused, isn't it?

Reply:
Yes. I moved from using this to adding the `createBuildFile` method in. Will remove

Reply:
removed